_sass/_utils/_colors.scss

```diff
@@ -0,0 +1,5 @@
+$green:             #24BA9D;
+$black:             #2B2B2B;
```

Comment:
@duranmla shall we call this grey instead?

Reply:
Are you sure? I mean I choose to not add fancy names to the colours because I understand that it could lead to more mess than solutions.

I am trying to use names that reflects the visual of the colour, when I see that colour it is very "blackish" and grey feels misleading.  

But I would like to understand your motivation to change to grey.

Reply:
Black and white are fixed values, #000000 for black and #FFFFFF, everything else in between is a darker or lighter grey. Not sure how having white as it’s value (FFFFFF) and black as a grey is less confusing